src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -132,9 +133,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         // load any messages from templates
         $extractedCatalogue = new MessageCatalogue($input->getArgument('locale'));
         $io->comment('Parsing templates...');
-        $prefix = $input->getOption('prefix');
         $extractor = $this->getContainer()->get('translation.extractor');
-        $extractor->setPrefix(null === $prefix ? '' : $prefix);
+        $extractor->setPrefix(false === $input->getOption('no-prefix') ? $input->getOption('prefix') : '');
```

Comment:
I think `$input->getOption('no-prefix') ? '' : $input->getOption('prefix')` will be more readable.